packages/core/src/_common/base/socket_base.js

```diff
@@ -249,6 +249,16 @@ const BinarySocketBase = (() => {
 
     const tncApproval = () =>
         deriv_api.send({ tnc_approval: '1' });
+    
+    const contractUpdate = (contract_id, req) => deriv_api.send({
+        contract_update: 1,
+        contract_id,
+        ...req,
+    });
+
+    const cancel = (contract_id) => deriv_api.send({
```

Comment:
nit; Change to `cancelContract`? To make it consistent with other `camelCase` function names.